src/sfeQwiicUltrasonic.h

```diff
@@ -16,8 +16,16 @@
 
 // Available I2C addresses of the Qwiic Ultrasonic
 const uint8_t kQwiicUltrasonicDefaultAddress = 0x2F;
-const uint8_t kQwiicUltrasonicMinAddress = 0x08;
-const uint8_t kQwiicUltrasonicMaxAddress = 0x7F;
+// These addresses are the min and max (respectively) of valid I2C addresses that can
+// be used for the newest revision of the Qwiic Ultrasonic sensor.
+const uint8_t kQwiicI2CAddressMin = 0x08;
+const uint8_t kQwiicI2CAddressMax = 0x7F;
+// Available I2C addresses of the Qwiic Ultrasonic
+const uint8_t kQwiicUltrasonicAddresses[] = {0x20, 0x21, 0x22, 0x23, 0x24, 0x25, 0x26, 0x27,
+                                             0x28, 0x29, 0x2A, 0x2B, 0x2C, 0x2D, 0x2E, 0x2F};
+const uint8_t kQwiicUltrasonicNumAddresses = sizeof(kQwiicUltrasonicAddresses) / sizeof(uint8_t);
+const uint8_t kQwiicUltrasonicMinAddress = kQwiicUltrasonicAddresses[0];
+const uint8_t kQwiicUltrasonicMaxAddress = kQwiicUltrasonicAddresses[15];;
```

Comment:
I think I wrote this originally, but honestly I don't think it makes sense to use an array for this, just the min and max address values.